samcli/lib/remote_invoke/remote_invoke_executors.py

```diff
@@ -25,15 +35,27 @@ class RemoteInvokeExecutionInfo:
     # Request related properties
     payload: Optional[Union[str, List, dict]]
     payload_file: Optional[TextIOWrapper]
+    parameters: dict
+    output_format: RemoteInvokeOutputFormat
 
     # Response related properties
     response: Optional[Union[dict, str]]
+    stderr_str: Optional[str]
     exception: Optional[Exception]
 
-    def __init__(self, payload: Optional[Union[str, List, dict]], payload_file: Optional[TextIOWrapper]):
+    def __init__(
+        self,
+        payload: Optional[Union[str, List, dict]],
+        payload_file: Optional[TextIOWrapper],
+        parameters: dict,
+        output_format: RemoteInvokeOutputFormat,
+    ):
         self.payload = payload
         self.payload_file = payload_file
+        self.parameters = parameters
+        self.output_format = output_format
         self.response = None
+        self.stderr_str = None
```

Comment:
Let's rename `stderr_str` to something else (like `log_output`)

Reply:
Addressed in the new commit.